AutoRest/Generators/CSharp/CSharp/Templates/MethodTemplate.cshtml

```diff
@@ -228,7 +228,7 @@ else
 
     HttpStatusCode _statusCode = _httpResponse.StatusCode;
     cancellationToken.ThrowIfCancellationRequested();
-    string _responseContent = null;
+    string _responseContent = await _httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
```

Comment:
What if the payload is a stream? You will end up with OutOfMemory exception for APIs that pipe back files or blobs.